acceptance_tests/features/add_a_survey.feature

```diff
@@ -46,4 +54,6 @@ Feature: Add a survey
   Scenario: User can cancel at any point
     Given the user has entered a valid enrolment code
     When they navigate to the confirm organisation page and click cancel
-    Then the user is navigated back to their "To do" list and they have not enrolled for that survey
\ No newline at end of file
+    Then the user is navigated back to their "To do" list and they have not enrolled for that survey
+
```

Comment:
It's not at all important here, but just be careful about accidentally adding whitespace